installers/npm/download.js

```diff
@@ -53,7 +53,7 @@ module.exports = function(callback)
 	});
 
 	// put it all together
-	request(url).on('error', reportDownloadFailure).pipe(gunzip).pipe(write);
+	request.get(url).then(gunzip).then(write).catch(reportDownloadFailure);
```

Comment:
When I run `node install.js` on master, it replaces `bin/elm` (a Node.js script) with an elm binary, which is the expected behavior.

On your branch, `node install.js` results in `bin/elm` becoming an empty file.

`.pipe` is for chaining streams, while `.then` is for chaining promises?